tests/test_data/test_pipelines/test_augmentation.py

```diff
@@ -364,18 +364,40 @@ def test_color_jitter(self):
 
         results = copy.deepcopy(self.results)
         results['gt'] = (results['gt'] * 255).astype(np.uint8)
+        results['lq'] = [results['gt'], results['gt']]
 
-        target_keys = ['gt']
+        target_keys = ['gt', 'lq']
 
         color_jitter = ColorJitter(
-            keys=['gt'], brightness=0.5, contrast=0.5, saturation=0.5, hue=0.5)
+            keys=['gt', 'lq'],
+            brightness=0.5,
+            contrast=0.5,
+            saturation=0.5,
+            hue=0.5)
+        color_jitter_results = color_jitter(results)
```

Comment:
This result is overwritten in line 385 before the assertion. We need to add some assert for this result.